openedx/core/djangoapps/content/course_structures/signals.py

```diff
@@ -10,4 +10,4 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
 
     # Note: The countdown=0 kwarg is set to to ensure the method below does not attempt to access the course
     # before the signal emitter has finished all operations. This is also necessary to ensure all tests pass.
-    update_course_structure.delay(unicode(course_key), countdown=0)
+    update_course_structure.apply_async((unicode(course_key),), countdown=0)
```

Comment:
Nit: I tend to prefer `[]` for single element list/tuples, because the trailing comma is easy to forget.